### ISSUE TITLE
Future results is not rendered

### ISSUE THREAD

**Issue by rb-de0** [vapor/template-kit#10]
I passed an encodable object to a renderer like a document, but Future's results are not displayed.

https://docs.vapor.codes/3.0/leaf/basics/#async

I looked at template-kit sources.
I think that it is caused by returning encoder.context in the following places. 

https://github.com/vapor/template-kit/blob/1.0.0-rc.2.0.3/Sources/TemplateKit/Data/TemplateDataEncoder.swift#L72

I think that by returning data at codingPath not context, ```fetch```  in TemplateContext can be processed correctly.

Please check this issue.

**Comment by 0xTim** [comment]
@rb-de0 can you show the code where you create a context and pass it to the renderer?

**Comment by rb-de0** [comment]
It is as follows.
<br>

## Code

Leaf Template (index.leaf)
```Leaf
<!DOCTYPE html>
<html>

<head>
  <title>test</title>
</head>

<body>
#(currentUser.name)
</body>

</html>
```

Models
```Swift
struct User: MySQLModel, Migration {
    var id: Int?
    var name: String
}

struct Profile: Encodable {
    var currentUser: Future<User>
}
```

Routing
```Swift
public func routes(_ router: Router) throws {   
    router.get("/") { req -> Future<View> in
        let currentUser = try User.find(1, on: req).unwrap(or: Abort(.internalServerError))
        let profile = Profile(currentUser: currentUser)
        return try req.make(LeafRenderer.self).render("index", profile)
    }
}
```

**Comment by rb-de0** [comment]
I'm sorry to bother you.
Are fixes for this issue included in next release?

@0xTim @tanner0101

**Comment by jdmcd** [comment]
I’m seeing this problem in my project too

**Comment by tanner0101** [comment]
Planning to look into this before the 1.0.0 tag 👍

**Comment by tanner0101** [comment]
This has been fixed in 1.0.0.
